### PR TITLE
Issue #1005 Interactive SSH

### PR DESCRIPTION
**Pull request by gbraad**
#1005

### CODE REVIEW

**Comment by gbraad** [comment]
This is just a quick and dirty fix to communicate and check if the functionality now works as expected accross the board

File: pkg/minikube/cluster/cluster.go [hunk 1]
```diff
@@ -508,7 +508,12 @@ func CreateSSHShell(api libmachine.API, args []string) error {
 	if err != nil {
 		return err
 	}
-	return client.Shell(strings.Join(args, " "))
+
+	command := strings.Join(args, " ")
```
Comment (budhrg) [inline pkg/minikube/cluster/cluster.go]: Awesome! Looks like we can contribute back to minikube :)

**Comment by budhrg** [comment]
Working fine for me :+1:

**Comment by LalatenduMohanty** [comment]
Now we wont have to be dependent of any SSH Client, which is really cool.

**Comment by hferentschik** [comment]
Merged, thanks.

**Pull request merged**